.github/workflows/ci.yml

```diff
@@ -128,9 +140,6 @@ jobs:
   openbsd:
     runs-on: ubuntu-latest # macos would be faster. TODO: diagnose why some tests are failing
     steps:
-      - uses: actions/checkout@v3
-        with:
-          submodules: true
```

Comment:
This change is causing the OpenBSD CI workflow to fail. OpenBSD doesn't ship git in the base install, however builder will try to do a git clone before it installs the platform packages. The workflow needs to check out the project in order to bootstrap builder.